### PR TITLE
OSSM-4778: Sample command line is missing closing parenthesis

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OSSM-4778

Link to docs preview: https://64260--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-observability#ossm-integrating-with-user-workload-monitoring_observability

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by rh-tokeefe** [comment]
@jewertow please provide dev review on this code change

File: modules/ossm-integrating-with-user-workload-monitoring.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ $ SECRET=`oc get secret -n openshift-user-workload-monitoring |
 +
 [source,terminal]
 ----
-$ TOKEN=`echo $(oc get secret $SECRET -n openshift-user-workload-monitoring -o jsonpath='{.data.token}' | base64 -d`
+$ TOKEN=`echo $(oc get secret $SECRET -n openshift-user-workload-monitoring -o jsonpath='{.data.token}') | base64 -d`
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-integrating-with-user-workload-monitoring.adoc]: Thanks @luksa! PR has been revised.

**Comment by luksa** [comment]
LGTM

**Comment by pbajjuri20** [comment]
LGTM..!

**Review by pbajjuri20 — APPROVED** [review]

**Review by bburt-rh — APPROVED** [review]

**Comment by bburt-rh** [comment]
lgtm

**Review by jeana-redhat — APPROVED** [review]
/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**